examples/full-config/wasmcloud-annotated.yaml

```diff
@@ -68,6 +68,9 @@ spec:
     traces:
       enable: false
       endpoint: "traces-specific-otel-collector.svc"
+  # Optional: Subject prefix that will be used by the host to query for wasmCloud Secrets.
+  # See https://wasmcloud.com/docs/concepts/secrets for more context
+  secrets_topic_prefix: "wasmcloud.secrets"
```

Comment:
heads up this will actually be `secretsTopicPrefix` due to camel-casing in the struct.